src/renderer/Layout.tsx

```diff
@@ -94,6 +95,13 @@ const tabProps = (index: number) => {
   }
 }
 
+/**
+ * Get video player settings initially when the component is loaded,
+ * because we do want to get it from here one time, but subsequently
+ * we'll only need to use the IPC channel `videoPlayerSettings`.
```

Comment:
Not sure I totally understand comment. This is hit whenever we render the component I think (i.e. change category or video) -- so the other than this we only use the IPC set method to tell main to remember the settings. Is that right?

Reply:
Honestly, I'm not sure why it works - I don't quite understand the React component lifecycle well, but it works.